0012-ledger-web-api/0012-ledger-web-api.md

```diff
@@ -0,0 +1,193 @@
+# Ledger Web API
+
+This spec defines some guidelines for **RESTful ledger APIs** to easily integrate with the Interledger Protocol suite.
+
+The advantages of following this spec when designing a ledger are:
+- A standardized interface for inter-ledger communication
+- Easy adoption of existing interledger plugins
+
+<p align="center">
+  <img width="70%" height="70%" src ="./static/ilp_interface.png" />
+</p>
+
+## Requirements
+
+In order for ledgers to connect seamlessly with any of the ILP plugins `ilp-plugin-<ledgerID>`, the ledger needs to provide:
```

Comment:
"In order for ledgers to integrate seamlessly with connectors the ledger needs to provide:"